### PR TITLE
Add deploy

### PR DESCRIPTION
**Pull request by karol-bisztyga**

### CODE REVIEW

**Comment by karol-bisztyga** [comment]
Test in protostar https://github.com/software-mansion/protostar/pull/1597

File: corelib/src/cheatcodes.cairo [hunk 1]
```diff
@@ -9,3 +9,7 @@ extern fn declare(contract: felt) -> Result::<felt, felt> nopanic;
 extern fn invoke(contract_address: felt, entry_point_selector: felt, calldata: Array::<felt>) -> Result::<(), felt> nopanic;
 
 extern fn mock_call(contract_address: felt, entry_point_selector: felt, response: Array::<felt>) -> Result::<(), felt> nopanic;
+
+// TODO prepared_contract to be a struct
+// returns deployed `contract_address`
+extern fn deploy(prepared_contract_address: felt, prepared_class_hash: felt, prepared_constructor_calldata: Array::<felt>) -> Result::<felt, felt> nopanic;
```
Comment (MaksymilianDemitraszek) [inline corelib/src/cheatcodes.cairo:15]: We could name it `deploy_tp` and write a wrapper in cairo which unpacks the structure

**Review by MaksymilianDemitraszek — APPROVED** [review]
If it works I am ok with merging it, can we add the wrapper before merging though?

File: corelib/src/cheatcodes.cairo (hunk 1 above)
Comment (karol-bisztyga) [reply corelib/src/cheatcodes.cairo]: I didn't really know what `tp` means here, I don't mind naming as usual but I added just a `wrapper` suffix. We can change this name anytime honestly.

**Pull request merged**